services/QuillLMS/client/app/bundles/Teacher/components/assignment_flow/create_unit/custom_activity_pack/filter_column.tsx

```diff
@@ -23,7 +23,7 @@ interface FilterColumnProps {
   activityClassificationFilters: string[],
   handleActivityClassificationFilterChange: (activityClassificationFilters: string[]) => void,
   ccssGradeLevelFilters: number[],
-  ellFilters: string[],
+  ellFilters: number[],
```

Comment:
See previous comment - I recommend CMD+F'ing your next PR revision to ensure that the ellFilters typing is all `string[]`